duckiebot/intrinsics_calibrator/command.py

```diff
@@ -0,0 +1,29 @@
+from dt_shell import DTCommandAbs, DTShell
+from utils.assets_utils import get_asset_icon_path
+from utils.duckietown_viewer_utils import \
+    ensure_duckietown_viewer_installed, launch_viewer
+
+# NOTE: this must match the name of the launcher in the dt-duckietown-viewer project
+LAUNCHER_NAME = "intrinsics_calibrator"
+ICON_ASSET = "icon-intrinsics-calibrator.png"
+
+class DTCommand(DTCommandAbs):
+    help = "Runs the intrinsics calibrator using the Duckietown Viewer app"
```

Comment:
We can remove the part "using the Duckietown Viewer app" as it repeats in every command and is not very useful to the user.